tests/monotouch-test/System.Net.Http/MessageHandlers.cs

```diff
@@ -110,5 +112,45 @@ public void TestNSUrlSessionHandlerCookies ()
 		}
 #endif
 
+		// ensure that if we have a redirect, we do not have the auth headers in the following requests
+#if !__WATCHOS__
+		[TestCase (typeof (HttpClientHandler))]
+		[TestCase (typeof (CFNetworkHandler))]
+#endif
+		[TestCase (typeof (NSUrlSessionHandler))]
+		public void RedirectionWithAuthorizationHeaders (Type handlerType)
+		{
```

Comment:
you need
```
			TestRuntime.AssertSystemVersion (PlatformName.MacOSX, 10, 9, throwIfOtherPlatform: false);
			TestRuntime.AssertSystemVersion (PlatformName.iOS, 7, 0, throwIfOtherPlatform: false);
```
so it can run on the oldest OS we support.

Maybe not the MacOS anymore, since I think 10.9 is the new minimum